asapdiscovery-data/asapdiscovery/data/state_expanders/expansion_tag.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+
+from pydantic import UUID4, BaseModel, Field
+
+
+class StateExpansionTag(BaseModel):
```

Comment:
It would be good to capture the provenance of the state enumeration here as well then we also know what state was enumerated.

Reply:
Agree, just trying to sort out some circular module nonsense first before I can put the expansion into the tag.